_posts/2023-09-25-announcing-duckdb-090.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+title:  "Announcing DuckDB 0.9.0"
+author: Mark Raasveldt and Hannes Mühleisen
+excerpt_separator: <!--more-->
+---
+
+<img src="/images/blog/yellow-billed-duck.jpg"
+     alt="Image of the Yellow Billed Duck"
+     width=200px
+     />
+
+The DuckDB team is happy to announce the latest DuckDB release (0.9.0). This release is named Undulata after the [Yellow-billed duck](https://en.wikipedia.org/wiki/Yellow-billed_duck) native to Africa.
+
+To install the new version, please visit the [installation guide](https://duckdb.org/docs/installation/index). The full release notes can be found [here](https://github.com/duckdb/duckdb/releases/tag/v0.9.0).
+
+<!--more-->
+
+#### What's new in 0.9.0
+
+There have been too many changes to discuss them each in detail, but we would like to highlight several particularly exciting features! 
+
+* Out-Of-Core Hash Aggregate
+* Storage Improvements
+* Index Improvements
+* DuckDB-WASM Extensions
+* Extension Auto-Loading
+* Improved AWS Support
+* Iceberg Support
+* Azure Support
+* PySpark-Compatible API
+
+Below is a summary of those new features with examples, starting with a change in our SQL dialect that is designed to produce more intuitive results by default.
+
+
+#### Breaking SQL Changes
+
+[**Struct Auto-Casting**](https://github.com/duckdb/duckdb/pull/8942). Previously the names of struct entries were ignored when determining auto-casting rules. As a result, struct field names could be silently renamed. Starting with this release, this will not throw an exception instead.
```

Comment:
```suggestion
[**Struct Auto-Casting**](https://github.com/duckdb/duckdb/pull/8942). Previously the names of struct entries were ignored when determining auto-casting rules. As a result, struct field names could be silently renamed. Starting with this release, this will now throw an exception instead.
```